NEXT-25815 - Recurring payments (#982)
shopware/docs: guides/plugins/apps/payment.md

You may choose between a synchronous and an asynchronous payment method. These two types are differentiated by defining a `finalize-url` or not. If no `finalize-url` is defined, the internal Shopware payment handler will default to a synchronous payment. If you do not want or need any communication during the payment process with your app, you can also choose not to provide a `pay-url`, then the payment will remain on open on checkout.

Below you can see different definitions of payment methods.

> Depending on what urls you provide, Shopware knows, which kind of payment flow your payment method supports.

Reviewer: ```suggestion
Depending on the URLs you provide, Shopware knows which kind of payment flow your payment method supports.
```

{% endtabs %}

## Recurring captures

{% hint style="info" %}
Recurring orders and payments require the Subscriptions feature, available exclusively in our [paid plans](https://www.shopware.com/en/pricing/).
{% endhint %}

> Recurring payments are a special case of payment that is used for handling recurring orders, such as subscriptions. The request and response payloads are similar to the synchronous payment flow. At this point, a valid running billing agreement between the customer and the PSP should exist. Use any of the other payment flows to capture the initial order and create such an agreement during the checkout. Afterwards, the payment can be captured via this flow for every recurring payment order.

Reviewer: ```suggestion
The recurring payment is a special payment used for handling recurring orders, such as subscriptions. It also follows request and response payloads similar to the synchronous payment flow. At this point, a valid running billing agreement between the customer and the PSP should exist. The initial preparation of the billing agreement (getting the customer approval & token)  should take place in known workflows like sync/async. This information can then be obtained from the recurring property of the transaction struct. Use any of the other payment flows to capture the initial order and create such an agreement during the checkout. Afterwards, the payment can be captured via this flow for every recurring payment order.
```

> Recurring payments are a special case of payment, which is used for handling recurring orders, such as subscriptions. The request and response payloads are similar to the synchronous payment flow. At this point, a valid running billing agreement between the customer and the PSP should exist. Use any of the other payment flows to capture the initial order and create such an agreement during the checkout. Afterwards, for every recurring payment order, the payment can be captured via this flow.

Reviewer: ```suggestion
Recurring payments are a special case of payment that is used for handling recurring orders, such as subscriptions. The request and response payloads are similar to the synchronous payment flow. At this point, a valid running billing agreement between the customer and the PSP should exist. Use any of the other payment flows to capture the initial order and create such an agreement during the checkout. Afterwards, the payment can be captured via this flow for every recurring payment order.
```
